walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -189,26 +235,46 @@ A *Path* represents an API endpoint/operation and the associated parameters requ
 
 . Click the orange *POST* icon.
```

Comment:
Looks like they changed the API Designer UI. Let's remove the orange reference and just say "under the operations heading, select POST"

Then we need to change the following to say "Next, click the `Add Operation` button". The carry on to the "Enter the following in the Summary field" bit...